src/main/kotlin/uk/gov/justice/digital/hmpps/prisonerfromnomismigration/alerts/AlertsSynchronisationService.kt

```diff
@@ -205,8 +207,10 @@ class AlertsSynchronisationService(
         mappingType = NOMIS_CREATED,
       )
     }
-
-    val mappingResponse = tryToCreateMappings(mappings, telemetry)
```

Comment:
delete mappings for the ones DPS told us were deleted

Reply:
Is there any value in keeping them so you have a history?

Reply:
Nah - it was just be a dangling record that will never be read